From d58958bf52a33b8727c5370728e83604e8d506a6 Mon Sep 17 00:00:00 2001
From: ZhongWen <b47623@freescale.com>
Date: Fri, 27 Sep 2013 16:58:25 +0800
Subject: [PATCH 29/52]  ENGR00278171 wfd-sink: update wfd-sink apk

    Do some optimization for the apk:
       1.change the refresh button

Signed-off-by: ZhongWen <b47623@freescale.com>
---
 WfdSink/res/layout/sink_main.xml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/WfdSink/res/layout/sink_main.xml b/WfdSink/res/layout/sink_main.xml
index bbe2827..b2fbd3f 100755
--- a/WfdSink/res/layout/sink_main.xml
+++ b/WfdSink/res/layout/sink_main.xml
@@ -17,8 +17,8 @@
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-	android:layout_gravity="center"
-	android:background="#000000"
+    android:layout_gravity="center"
+    android:background="#000000"
     android:orientation="vertical" >
 
     <RelativeLayout
-- 
1.8.0

